src/main/resources/com/puppycrawl/tools/checkstyle/suppressions_1_2.dtd

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Add the following to any file that is to be validated against this DTD:
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.2//EN"
+    "http://checkstyle.sourceforge.net/dtds/suppressions_1_2.dtd">
+-->
+
+<!ELEMENT suppressions (suppress*)>
+
+<!ELEMENT suppress EMPTY>
+<!ATTLIST suppress files CDATA #IMPLIED
+                   checks CDATA #IMPLIED
+                   messages CDATA #IMPLIED
```

Comment:
should be "message" as it is not a list

Reply:
@romani I made it plural as files and checks is plural and they both do the same thing, a regular expression match. Even lines and columns is plural. Only id is currently single but I don't know why.
Do you still want it singular?

Reply:
there is expression and it is one .... fact that it matching for many is files/Checks or any other content is a different story.
lines/columns are plural because of ranges and lists.

Lets not follow bad naming here, fortunately "id" was named correctly.

Reply:
Done.